src/main/kotlin/org/rust/cargo/project/configurable/RustProjectConfigurable.kt

```diff
@@ -97,6 +113,8 @@ class RustProjectConfigurable(
         useCargoCheckAnnotator = settings.useCargoCheckAnnotator
         letBindingHint = HintType.LET_BINDING_HINT.enabled
         parameterHint = HintType.PARAMETER_HINT.enabled
+        lambdaParameterHint = HintType.LAMBDA_PARAMETER_HINT.enabled
+        smartHint = HintType.SMART_HINTING.get()
```

Comment:
At the moment, we have several checkboxes in the Rust settings panel, and they use three different approaches to getting/setting their values. Is this variety necessary?